src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double length;
+    private double width;
+
+    public Rectangle(String color, double length, double width) {
+        super(color);
+        this.length = length;
+        this.width = width;
+    }
+
+    @Override
+    public double area() {
+        return length * width;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Rectangle color: " + getColor()
+                + ", Length = " + length + ", Width = " + width
+                + "Area = " + area());
```

Comment:
There is a missing comma in the 'System.out.println' statement between the width and the area. Also, consider using 'System.out.format' for better readability and string formatting.